utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -135,157 +141,116 @@ TEST_P(WriteUnpreparedTransactionTest, ReadYourOwnWrite) {
   }
 }
 
-#if !defined(ROCKSDB_VALGRIND_RUN) || defined(ROCKSDB_FULL_VALGRIND_RUN)
-TEST_P(WriteUnpreparedStressTest, ReadYourOwnWriteStress) {
-  // This is a stress test where different threads are writing random keys, and
-  // then before committing or aborting the transaction, it validates to see
-  // that it can read the keys it wrote, and the keys it did not write respect
-  // the snapshot. To avoid row lock contention (and simply stressing the
-  // locking system), each thread is mostly only writing to its own set of keys.
+TEST_P(WriteUnpreparedSnapshotTest, ReadYourOwnWrite) {
+  // This test validates a transaction can read its writes and the correctness
+  // of its read with regard to a mocked snapshot functionality.
```

Comment:
Yes, this removed testing of the unowned keys, the keys that are not written by the transaction. Mostly because as part of removing the stressing aspect of this unit test, it was transformed from a multi threaded test to a single thread test. Previously each thread has a transaction that does writing to its owned keys and randomly read a small portion of unowned keys. I think in that setting, it makes sense to read other keys because presumably other threads and other transactions will attempt to write to those keys, and we can test those thread's writes are not leaked to other threads.

Since it's just a single thread test now, I think it's not as interesting to verify keys that are not written. I think the part for testing "cannot read other transaction's write" should be covered by the stress test now.